containerfiles/unstable

```diff
@@ -0,0 +1,136 @@
+FROM quay.io/fedora/fedora-bootc@sha256:b11b87e0f1d843c14e7458b5d12eb70682c9a2a63eb46b6592bceeddd1a40162
```

Comment:
This exact container no longer exists. We should either (1) keep a copy of this on GitHub Container Registry or (2) use the `:latest` tag.